lib/moreInfoService.php

```diff
@@ -86,8 +86,7 @@ public function getByFaustNumber($faustNumber) {
    *   Array of the images that were found.
    */
   public function getByLocalIdentifier($local_id) {
-    $identifiers = $this->collectIdentifiers('localIdentifier', $local_id);
-    $response    = $this->sendRequest($identifiers);
+    $response    = $this->sendRequest($local_id);
 
     if(empty($response->identifierInformation)){
```

Comment:
Kan du ikke løbe koden igennem med kodestandarden? Så vi får mellemrum de rigtige steder osv.